### PR TITLE
feat: add a stage to the config to track the output dataset IDs

### PR DESCRIPTION
**Pull request by jrs65**

### CODE REVIEW

File: config/chime_science_run_recv.yaml [hunk 1]
```diff
@@ -267,6 +266,28 @@ remove_ev:
...
+    in_buf: visbuf_5s_26m_ungated_post
+  26mgated:
+    kotekan_stage: visDebug
+    in_buf: visbuf_5s_26m_gated_post
+  cal:
+    kotekan_stage: visDebug
+    in_buf: visbuf_10s_cal
+  timing:
+    kotekan_stage: visDebug
+    in_buf: visbuf_10s_timing_ne{
```
Comment (nritsche) [inline config/chime_science_run_recv.yaml]:
```suggestion
    in_buf: visbuf_10s_timing_ne
```

Comment (andrerenard) [inline config/chime_science_run_recv.yaml]:
```suggestion
    in_buf: visbuf_10s_timing_ne
```

Comment (jrs65) [reply config/chime_science_run_recv.yaml]: Ooops. Stupid vim.

Comment (jrs65) [reply config/chime_science_run_recv.yaml]: I think you and @nritsche both caught the same thing. It's vim's fault.

**Pull request merged**